resources/schema/utils.py

```diff
@@ -42,7 +42,9 @@ def resolve_piers(info, **kwargs):
             )
 
             min_width = application.boat_width
-            min_length = application.boat_length
+
+            # VEN-1251: berths that are too short need to be shown in the UI
+            min_length = None
```

Comment:
Is it necessary to explicitly set it as `None` or would the declaration `min_length = kwargs.get("min_berth_length")` be enough?

Reply:
hmm, that would work too. fixed.